package.json

```diff
@@ -34,11 +34,10 @@
     "react-ga4": "1.4.0",
     "react-redux": "7.2.4",
     "react-router-dom": "5.2.1",
-    "react-scripts": "4.0.3",
     "reactstrap": "8.9.0",
     "redux": "4.1.1",
     "redux-thunk": "2.3.0",
-    "vitessce": "1.1.21"
+    "sass": "1.56.1"
```

Comment:
node-sass-chokidar is now deprecated, and the recommendation is to switch over to dart-sass.